test/runtests.jl

```diff
@@ -484,10 +484,7 @@ end
             @test_throws Exception fits_read_pix(f, a)
             @test_throws Exception fits_read_pix(f, a, 1)
             @test_throws Exception fits_read_pixnull(f, a, similar(a, UInt8))
-
-            # write some data to avoid errors on closing
-            fits_create_img(f, eltype(a), [size(a)...])
-            fits_write_pix(f, a)
+            @test_throws Exception fits_write_pix(f, a)
```

Comment:
`Exception` is extremely generic, can you check a more concrete error type?

Reply:
I've changed this to `ErrorException`. The idea initially was to not be too strict on the type of the exception just in case this is changed later